src/types/zodFlaw.ts

```diff
@@ -322,6 +322,22 @@ export const ZodFlawSchema = z.object({
     return null;
   };
 
+  if (
+    zodFlaw.requires_summary
+    && ['REQUESTED', 'APPROVED'].includes(zodFlaw.requires_summary)
+    && zodFlaw.summary === ''
+  ) {
+    raiseIssue('Description cannot be blank if requested or approved.', ['summary']);
+  }
+
+  if (
+    zodFlaw.requires_summary !== 'APPROVED'
+    && zodFlaw.major_incident_state
+    && ['APPROVED', 'CISA_APPROVED'].includes(zodFlaw.major_incident_state)
```

Comment:
Same thought as previous:
```suggestion
    zodFlaw.requires_summary !== 'APPROVED'
    && ['APPROVED', 'CISA_APPROVED'].includes(zodFlaw.major_incident_state)
```